engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportToSizeOf.java

```diff
@@ -95,7 +81,6 @@ public String toString() { // TODO: used for logging purposes at the moment, inv
     }
 
     private FBO getFbo() {
-        return defaultDynamicFBO != null ? defaultDynamicFBO.getFbo() : frameBuffersManager.get(fboName);
-
+        return frameBuffersManager.get(fboName);
```

Comment:
I'd be happy if this subscribes to the fboManager. As it is a state change it doesn't have to be lean.

But where are the calls you are referring to?